modules/nodes-descheduler-configuring-other-settings.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-configuring-other-settings_{context}"]
+= Configuring additional descheduler settings
+
+You can configure additional settings for the descheduler, such as how frequently it runs.
+
+.Prerequisites
+* Cluster administrator privileges.
+
+.Procedure
+
+. Edit the `KubeDescheduler` object:
++
+----
+$ oc edit kubedeschedulers.operator.openshift.io cluster -n openshift-kube-descheduler-operator
+----
+
+. Configure additional settings as necessary:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1beta1
+kind: KubeDescheduler
+metadata:
+  name: cluster
+  namespace: openshift-kube-descheduler-operator
+spec:
+  deschedulingIntervalSeconds: 3600 <1>
+  flags:
+  - --dry-run <2>
+  image: quay.io/openshift/origin-descheduler:4.4 <3>
```

Comment:
@damemi @ingvagabund I was looking for an example to put for the image field. Is this the correct default image going to be used for 4.4?

And can you also verify that the format of the flags field is correct? It seems to accept it like this, but just wanted to be sure since I didn't see an example anywhere.